code/datums/ammo/bullet/pistol.dm

```diff
@@ -167,8 +166,6 @@
 
 /datum/ammo/bullet/pistol/highpower
 	name = "high-powered pistol bullet"
-	headshot_state = HEADSHOT_OVERLAY_MEDIUM
```

Comment:
why remove the overlays?

Reply:
because it is already set the same way  in is parent /datum/ammo/bullet/pistol